### DISCUSSION TITLE
clarification on Frigate's birdseye functionality

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [blakeblackshear/frigate#15642] [Ask A Question]
The documentation says: "Birdseye can be viewed by adding the 'Birdseye' camera to a Camera Group in the Web UI. Add a Camera Group by pressing the '+' icon on the Live page, and choose 'Birdseye' as one of the cameras."

What does adding "Birdseye" to a camera group do?

Suppose I have two cameras in the front of my house which are in one camera group and two more cameras in the back of my house in a separate group. If I add "Birdseye" to both groups, what does that do? Can I set up Birdseye so that a camera group appears when Frigate detects an object in either of the cameras in that group?

**Comment by hawkeye217** [comment]
Birdseye is the original way that Frigate showed you live streams of your cameras before the changes in 0.14.

The docs are just describing the method to view the Birdseye camera. There is only one Birdseye camera, which you can customize [per the docs](https://docs.frigate.video/configuration/birdseye/).
